src/CollAction/Helpers/ProjectBannerImageManager.cs

```diff
@@ -0,0 +1,68 @@
+﻿using CollAction.Models;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace CollAction.Helpers
+{
+    public class ProjectBannerImageManager : IImageFileManager
```

Comment:
It would probably be better to combine ProjectBannerImageManager, IImageFileManager and IFileManager in one class for now. This is probably another case of slight premature abstraction. Since we don't known if we're going to have multiple file managers, or multiple image file managers, it seems a bit much to use double inheritance, makes the code rather hard to follow. Maybe have some appropiate constructor parameters for the class, that make it generic to use (like begin able to customize the root directory where the images are stored). Also, https://en.wikipedia.org/wiki/Composition_over_inheritance.